invenio_drafts_resources/services/records/service.py

```diff
@@ -65,6 +66,7 @@ def search_drafts(self, identity, params=None, links_config=None,
             links_config=links_config,
             es_preference=es_preference,
             record_cls=self.draft_cls,
+            extra_filter=Q('term', **{'has_draft': False})
```

Comment:
Question: It's not ``has_draft==True`` we want?

Reply:
No because `has_draft` is set to False for the published records and the drafts....True is only for the published records with an active draft...but these records we get from the drafts index and we don't want to include them in the published ones...